### DISCUSSION TITLE
Is there any reason why AuthenticationMiddleware.Schemes property is publicly settable?

### DISCUSSION THREAD

**Discussion by voroninp** [dotnet/aspnetcore#58256] [Q&A]
I'm just curious why `Schemes` property is [declared as public settable](https://github.com/dotnet/aspnetcore/blob/6a40a23cd9242fc7ea01a9ae4f58886068f7257c/src/Security/Authentication/Core/src/AuthenticationMiddleware.cs#L34).

**Comment by voroninp** [comment]
@davidfowl, you are the arch, you know =)

**Reply by davidfowl** [reply]
![IjustworkhereSassyGIF](https://github.com/user-attachments/assets/5d62fd60-d2b1-4c13-a544-20ee490df08c)
